Dockerfile

```diff
@@ -10,9 +10,9 @@ ADD src ./src
 
 RUN mvn verify -T1C
 
-FROM sonatype/nexus3:3.19.1
+FROM sonatype/nexus3:3.21.2
 
-ARG NEXUS_CASC_VERSION=3.19.1-01
+ARG NEXUS_VERSION=3.21.2-03
```

Comment:
I think it should remain `NEXUS_CASC_VERSION` here. We can probably introduce another ARG for `NEXUS_VERSION` just before the FROM statement.

Reply:
Good spot, thank you - fixed!